torch/fx/symbolic_trace.py

```diff
@@ -449,4 +449,6 @@ def symbolic_trace(root : Union[torch.nn.Module, Callable]) -> GraphModule:
         GraphModule: a Module created from the recorded operations from ``root``.
 
     """
-    return GraphModule(root if isinstance(root, torch.nn.Module) else torch.nn.Module(), Tracer().trace(root))
+    return GraphModule(root if isinstance(root, torch.nn.Module) else torch.nn.Module(),
+                       Tracer().trace(root),
+                       root.__class__.__name__)
```

Comment:
In the case that `root` is a function, I think it makes more sense to pass `root.__name__` here:

```

def my_cool_fn(x):
    return x

print(my_cool_fn.__class__.__name__)
"""
function
"""

print(my_cool_fn.__name__)
"""
my_cool_fn
"""
```